cli-helpers/src/commands/invoke-avl-siri-vm-downloader.ts

```diff
@@ -13,10 +13,14 @@ export const invokeAvlSiriVmDownloader = new Command("invoke-avl-siri-vm-downloa
     .option("--originRef <originRef>", "Pass originRef parameter to function")
     .option("--destinationRef <destinationRef>", "Pass destinationRef parameter to function")
     .option("--subscriptionId <subscriptionId>", "Pass subscriptionId parameter to function")
+    .option("--apiKey <apiKey>", "Pass apiKey parameter to function")
```

Comment:
For the cli helpers could we go and retrieve the api key for each stage instead of requiring it as an input that the user has to pass through? Just thinking it might make the dev experience a bit nicer instead of having to go and fetch the api key each time we want to run this command